modules/standard/IO.chpl

```diff
@@ -3981,8 +3980,10 @@ proc channel.writeln(out error:syserr):bool {
 
 // documented in style= error= version
 pragma "no doc"
-proc channel.writeln():bool {
-  return this.write(new ioNewline());
+proc channel.writeln():bool throws {
+  try {
+    return this.write(new ioNewline());
+  }
```

Comment:
In Swift you can write such a method like this:
``` swift

func f() throws {
    return try g();
}
```

Is there a reason we can't support the same? (can you make a note in our error handling TODOs issue?)